src/client/fuse_ll.cc

```diff
@@ -1605,9 +1605,10 @@ int CephFuse::Handle::loop()
   if (fuse_multithreaded) {
 #if FUSE_VERSION >= FUSE_MAKE_VERSION(3, 1)
     {
-      struct fuse_loop_config conf = { 0 };
-
-      conf.clone_fd = opts.clone_fd;
+      struct fuse_loop_config conf = {
+        clone_fd: opts.clone_fd,
+        max_idle_threads: opts.max_idle_threads,
```

Comment:
nit: there are only two fields in `fuse_loop_config`, maybe the `,`(comma) here can be removed.

Reply:
OK, I will remove this comma. Thanks!